h2o-algos/src/main/java/hex/tree/SharedTree.java

```diff
@@ -159,6 +161,7 @@ protected void ignoreInvalidColumns(int npredictors, boolean expensive) {
     if (_parms._min_rows <=0) error ("_min_rows", "_min_rows must be > 0.");
     if (_parms._r2_stopping!=Double.MAX_VALUE) warn("_r2_stopping", "_r2_stopping is no longer supported - please use stopping_rounds, stopping_metric and stopping_tolerance instead.");
     if (_parms._score_tree_interval < 0) error ("_score_tree_interval", "_score_tree_interval must be >= 0.");
+    if (_parms._in_training_checkpoints_tree_interval <= 0) error ("_in_training_checkpoints_tree_interval", "_in_training_checkpoints_tree_interval must be >= 0.");
```

Comment:
also the reason why it doesn't default to `exports_checkpoint_dir` is because we don't want to trigger this in-training logic if user only provides `exports_checkpoint_dir`, right?

Reply:
> also the reason why it doesn't default to exports_checkpoint_dir is because we don't want to trigger this in-training logic if user only provides exports_checkpoint_dir, right?

yes,

Good point with reachable dir check, thank you!